### ISSUE TITLE
Is this still alive

### ISSUE THREAD

**Issue by MoBadawy** [epsylon3/torrentflux#59]
Hi, I can't install the script when I go to /setup.php it told me :

Internal Server Error

The server encountered an internal error or misconfiguration and was unable to complete your request.

Please contact the server administrator, webmaster@****************** and inform them of the time the error occurred, and anything you might have done that may have caused the error.

More information about this error may be available in the server error log.

Additionally, a 404 Not Found error was encountered while trying to use an ErrorDocument to handle the request.

I ask if this script still works?

**Comment by schmatzler** [comment]
Yes it still works, I have it running with MariaDB on Debian 7 here.
